js/client/modules/@arangodb/testsuites/authentication.js

```diff
@@ -182,10 +182,6 @@ function authenticationParameters (options) {
 
   let continueTesting = true;
   let results = {};
-  // we append one cleanup directory for the invoking logic...
-  let dummyDir = fs.join(fs.getTempPath(), 'authentication_dummy');
-  fs.makeDirectory(dummyDir);
-  pu.cleanupDBDirectoriesAppend(dummyDir);
```

Comment:
Don't we need to create `authentication_dummy` using `tmpDirManager` to replace the deteted code above?

Reply:
this is done by the instance manager inside the loop for each test:
```
    let instanceManager = new im.instanceManager('tcp', options,
                                                 authTestServerParams[test],
                                                 'authentication_parameters_' + authTestNames[test]);
```